README.md

```diff
@@ -13,21 +13,55 @@ Students can view their own and of their collegues time spent in a Bar Chart
 
 ## Requirements
 
-- Moodle Version: 4.1+
-- PHP Version: 7.4+
-- Supported Databases: MariaDB, PostgreSQL
-- Supported Moodle Themes: Boost
+- Supported Moodle Version: 4.1 - 4.5
+- Supported PHP Version:    7.4 - 8.2
```

Comment:
do we test against PHP 8.3 as well in yml file? If yes, I would specify 7.4 - 8.3 here